# STTA 254 (WZ Cas) - WDS - Cas

# Astrométrie et informations générales

In [10]:
import sys, os
# insérer le chemin suivant dans sys.path pour trouver le package astrodm et charger les outils
if 'D:\DOCUMENTS\Astronomie\dev' not in sys.path:
    sys.path.insert(0, 'D:\DOCUMENTS\Astronomie\dev')
from astrodm import doublesOutils as do


In [11]:
print(os.getcwd())

d:\DOCUMENTS\Astronomie\dev\Cahier Jupyter


In [9]:
obj_sys = do.Systeme(chemin_systeme= r"D:\DOCUMENTS\Astronomie\dev\med\STTA254")
print(obj_sys.informations_df)

  id_system      id_WDS      RAJ2000      DEJ2000 id_system_alt1 id_system_alt2 const  \
0   STTA254  00013+6021  00 01 15.85  +60 21 19.0     "A=WZ Cas"   "rechercher"   Cas   

          remarques  
0  "info du WDS ok"  


### Données du Washington Double Stars Catalog

In [2]:
do.info_src_wds('stta254', '*', True) # le paramètre True = imprimer les notes WDS.


Il y a 4 résultats pour «STTA254», paire = «*».

Données tirées du Washington Double Stars Catalog (WDS)
Tris sur "Disc" et "Comp"
------------------------------------------------------------------------------------------------------------------------
   WDS       Disc  Comp Obs1 Obs2 pa1 pa2  sep1   sep2   mag1   mag2    DM    Notes n_RAJ2000   RAJ2000     DEJ2000  
                         yr   yr  deg deg arcsec arcsec  mag    mag                             "h:m:s"     "d:m:s"  
---------- ------- ---- ---- ---- --- --- ------ ------ ------ ----- -------- ----- --------- ----------- -----------
00013+6021 STTA254   AB 1874 2019  90  89   58.9  57.80  7.400  8.33 +59 2810     N           00 01 15.85 +60 21 19.0
00013+6021 STTA254   AC 1879 2015 324 324  155.7 155.40  7.400  9.56 +59 2809     N           00 01 15.85 +60 21 19.0
00013+6021 STTA254   AD 1982 2016 118 118  182.0 181.40  7.400 10.35              N           00 01 15.85 +60 21 19.0
00013+6021 STTA254   BD 1879 2015 129 1

### Historique du système

Dans le WDS, la première données remonte à 1874 ... rechercher dans :
 - livre AVDS
 - CDSA
 - site de Webb
 - autre

## Instrumentation f/20 en «Lucky Imaging»

L'instrumentation pour le «Lucky Imaging» sur EdgeHD 8 est : ASI 120MM-S + Barlow 2X = f/20 appr. Le champ de la caméra est alors de 160 X 120 arcsec environ.

## Discussion sur les données du catalogue WDS
D'après les données de catalogue ci haut, la paire AB avec sep2 = 57.8 arcsec, semble parfaite pour l'instrumentation f/20 et sera observée en premier. A la rigueur, la paire BD (sep2 = 133.70 arcsec) et la paire AC (sep2 = 155.7 arcsec) pouraient tout juste être observées à f/20. Par contre, la paire AD (sep2 = 181.4 arcsec) est trop espacée pour cette instrumentation, mais pourrait éventuellement être mesurée sans la Barlow donc à f/10 environ.

L'instrumentation à f/10 reste à développer.

## Observations dans le cadre du programme «2021-023  - Réglages caméras pour le «Lucky Imaging» (LI)»

Dans le programme 2021-023, j'ai travaillé au développement du flux de travail pour le LI. Lors de la session d'observation 9 j'ai effectué le premier essai de la technique de calibration de l'échelle de la caméra en utilisant le masque à fentes de 3 mm et le filtre Ha. Par la suite, j'ai acquis des images pour mesurer la paire AB. Je n'ai cependant pas acquis des images de calibration photographiques (darks, etc.).

## Paire AB - résultats d'observation

In [3]:
do.reduction('complete', 'D:\DOCUMENTS\Astronomie\dev\med\STTA254\AB\P2021-023')



LISTE DES OBSERVATIONS
RÉDUCTION COMPLÈTE
----------------------------------------------------------------------------------------------------
id_system     id_WDS     RAJ2000     DEJ2000 id_system_alt1 id_system_alt2 const        remarques
  STTA254 00013+6021 00 01 15.85 +60 21 19.0     "A=WZ Cas"   "rechercher"   Cas "info du WDS ok"
----------------------------------------------------------------------------------------------------
    Paire : AB
Programme : P2021-023
----------------------------------------------------------------------------------------------------
   N bloc               obs_DH_UTC  obs_epoque_B  theta  theta_sigma      rho  rho_sigma  temp_to  \
0  1    1  2021-10-09T00:29:59.886     2021.7718  89.54         0.06  56.4697     0.2523     10.5   
1  1    2  2021-10-09T00:39:59.886     2021.7718  89.50         0.06  56.4726     0.2528     10.7   
2  2    1  2021-10-16T02:44:39.845     2021.7912  89.54         0.06  56.4408     0.3515     10.5   
3  2    2  2021-